v2/controllers/marketplace/marketplaceconfig_controller.go

```diff
@@ -71,6 +74,21 @@ type MarketplaceConfigReconciler struct {
 	cfg    config.OperatorConfig
 }
 
+// +kubebuilder:rbac:groups="",resources=namespaces,verbs=get;list;watch;update;patch
+// +kubebuilder:rbac:groups="",resources=secret,verbs=get;list;watch
```

Comment:
can the cluster-wide access to Secrets be restricted to specific namespace or specific Secret

Reply:
The operator manager is set to cluster scope, since there are several resources that must be watched at that scope.
The manager's cache function will throw errors if it does not have cluster get;list;watch on any resources types we watch, or otherwise do structured interaction with.